packages/ui-components/src/Avatar/Avatar.module.scss

```diff
@@ -0,0 +1,98 @@
+@import "../styles/tokens.scss";
+
+@mixin selectable-border($n) {
+  box-sizing: border-box;
+  border: 2px solid $n;
+}
+
+.avatar {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  font-family: Source Sans Pro, sans-serif; // TODO (koorosh): replace with fonts from tokes
+  font-style: normal;
+  font-weight: 600;
+  line-height: 1;
+  border-radius: 50%;
+  overflow: hidden;
+  user-select: none;
+}
+
+.intent-default {
+  background: $crl-neutral-2;
+  color: $crl-neutral-7;
+}
+
+.intent-info {
+  background: $crl-blue-1;
+  color: $crl-blue-3;
+}
+
+.intent-warning {
+  background: $crl-yellow-1;
+  color: $crl-yellow-4;
+}
+
+.intent-danger {
+  background: $crl-red-1;
+  color: $crl-red-4;
+}
+
+.size-40 {
+  width: 40px;
+  height: 40px;
+  font-size: 14px;
+  letter-spacing: 0.1px;
+}
+
+.size-32 {
+  width: 32px;
+  height: 32px;
+  font-size: 12px;
+  letter-spacing: 0.3px;
+}
+
+.size-24 {
+  width: 24px;
+  height: 24px;
+  font-size: 14px;
+  letter-spacing: 0.1px;
+}
+
+.size-16 {
+  width: 16px;
+  height: 16px;
+  font-size: 10px;
+  letter-spacing: 0.1px;
+}
+
+.disabled {
+  background: $crl-neutral-2;
+  color: $crl-neutral-5;
+
+  &:focus, &:hover {
+    border: none;
+  }
+}
+
+.selectable:not(.disabled) {
+  &:focus, &:hover {
+    &.intent-default {
+      @include selectable-border($crl-neutral-7);
+    }
+
+    &.intent-info {
+      @include selectable-border($crl-blue-3);
+    }
+
+    &.intent-warning {
+      @include selectable-border($crl-yellow-4);
+    }
+
+    &.intent-danger {
+      @include selectable-border($crl-red-4);
```

Comment:
Our warning color is `$crl-base-red`/`$crl-red-3`; comment re: base colors from above applies here too.